solutions/src/main/java/ru/sberbank/school/task07/DifferentWordsImpl.java

```diff
@@ -0,0 +1,32 @@
+package ru.sberbank.school.task07;
+
+import java.io.FileNotFoundException;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class DifferentWordsImpl implements DifferentWords {
+
+    private final FileParser fileParser;
+
+    public DifferentWordsImpl(FileParser fileParser) {
+        this.fileParser = fileParser;
+    }
+
+
+    @Override
+    public Object findSortedDifferentWords(String pathToFile) throws FileNotFoundException {
+        List<String> list = fileParser.parse(pathToFile);
+        return list.stream()
+                .map(s -> s.split("[^a-zA-Z]"))
+                .flatMap(Arrays::stream)
+                .filter(strings -> !strings.equals(""))
+                .sorted(((o1, o2) -> {if (o1.length() == o2.length()) {
+                    return o1.compareTo(o2);
+                }
+                    return o1.length() - o2.length();}))
```

Comment:
WhitespaceAround: &#39;}&#39; is not preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-600379303>*